meetings/2024-02-14-PAS-call-minutes.md

```diff
@@ -0,0 +1,182 @@
+
+# Protected Auction Services WICG Calls: Agenda & Notes
+
+Calls take place on some Wednesdays, at noon New York = 9am California.  Please check https://github.com/WICG/protected-auction-services-discussion/issues/27 for details.
+
+That's 6pm Paris time and 5pm UTC (those change sometimes, when some country fiddles with their clocks).
+
+(This notes doc will be editable by everyone, during the meeting)
+
+**Wednesday 14th February, 2024**
+
+Google Meet joining info
+
+Video call link: https://meet.google.com/htk-obgg-smb 
+
+Or dial: ‪(US) +1 509-676-3166‬ PIN: ‪117 463 428‬#
```

Comment:
GH says: "This file contains bidirectional Unicode text that may be interpreted or compiled differently than what appears below."